### PR TITLE
fix api docs generation

### PR DESCRIPTION
**Pull request by thimios**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have authority to submit this code.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: common/spec/lib/factory_bot_helpers.rb [hunk 1]
```diff
@@ -576,7 +576,7 @@ def few_or_none(key)
   end
 
   factory :json_bulk_archival_object_updater_job, class: JSONModel(:bulk_archival_object_updater_job) do
-    source { generate(:alphanumstr) }
+    create_missing_top_containers { false }
```
Comment (brianzelip) [inline common/spec/lib/factory_bot_helpers.rb:579]: @thimios is this related to fixing the api docs generation?

Comment (thimios) [reply common/spec/lib/factory_bot_helpers.rb:579]: @brianzelip this factory was buggy (there is no source property in the bulk archival object update job). The factory is not used in any spec though so we did not realize that it is buggy until I tried to generate the api docs, where all these factories are actually called:
https://github.com/archivesspace/archivesspace/blob/master/docs/api_doc.rb#L116
